### PR TITLE
Decode a Morse code message

### PR DESCRIPTION
**Pull request by diegoyon**
### Decode a Morse code message

- Created a method to decode a morse character
- Created a method to decode an entire word in morse
- Created a method to decode an entire sentence

### CODE REVIEW

File: morse_code.rb [hunk 1]
```diff
@@ -1 +1,57 @@
...
+decode_word '-- -.--'
+puts ' '
+
+def decode(sentence)
+  sentences = sentence.split('   ')
+  sentences.each do |n|
+    decode_word(n)
+    print ' '
+  end
+end
```
Comment (KDlamini) [inline morse_code.rb]: When I test your methods with the following morse code test inputs as per project requirements:

```
p decode_char('.-')
p decode_word('-- -.--')
p decode("-- -.--   -. .- -- .")
p decode(".-   -... --- -..-   ..-. ..- .-.. .-..    --- ..-.    .-. ..- -... .. . ...")
```

The expected output should be:

image

However, this is your output:

image

- Kindly look into this and make sure that all your methods output the desired outcome.

NB: Even when I run the test code without the `puts` shorthand `p` I still get an undesired output.

image

Output:

image

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi team @diegoyon, @Alzubair98 🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: morse_code.rb (hunk 1 above)
Comment (Alzubair98) [reply morse_code.rb]: ruby

Comment (Alzubair98) [reply morse_code.rb]: dear @KDlamini  thank you for your good review 
but i dont know what is that happening with you any way i will try to fix it

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED ✔️ 🟢 

## Hi team 👋, 

Well done for implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**